bartop-ui/.storybook/config.js

```diff
@@ -1,6 +1,8 @@
 import { configure, addDecorator } from '@storybook/react';
 import centered from '@storybook/addon-centered';
 
+import '../src/globals.css';
```

Comment:
Moved our global styles into its own stylesheet so that it can be shared between the actual UI & the storybook.